### DISCUSSION TITLE
DJ mode

### DISCUSSION THREAD

**Discussion by stoneubi** [fast4x/RiMusic#1748] [Ideas]
First I would like to say that this app is totally amazing. I recently looked for app being able to play music from YouTube without advertising and to use it for DJ modus.

So thank you very much for developing and creating this amazing app. 

Imagine you are at the party and you use your app at the tablet as a DJ and to get some wishes from the party guests to play a special song. 

You search for the song and once you find the song you always have to get into the menu and select que for next song. You have to be very careful otherwise you would change your entire playlist.

Feature request: 
It would be cool if have an option to switch on DJ modus. Having DJ modus enabled once you search for a song and you simply press on it it will add it to the playlist and will select it immediately to play it for the next song. This would help to keep the playlist safe.

Thank you very much 
Stoneubi

**Comment by fast4x** [comment]
Ahahaha amazing thanks for appreciations.
It looks interesting, I try to develop it.

One question: 
How would you like to turn on DJ mode? Specify exactly how you would like to use RiMusic to turn on DJ mode.

Please create a feature request ticket with the news I asked for. Thank you for your appreciation.
